### ISSUE TITLE
[Bug] Block HitBox removes smoke overlay

### ISSUE THREAD

**Issue by StupidRepo** [SongXia-NYU/nuclear_craft#14]
So, if you throw a smoke c4 down, it makes a smoke cloud right? Well, find some grass and make the outline square, that you can see on the outside of the grass, go onto of the overlay and BOOM, you can see through the smoke. It's hard to explain so hopefully, you understand.

**Comment by SongXia-NYU** [comment]
Yeah. I guess it is a bug from Minecraft that you can see through particles when inside of some hitbox. Hopefully Mojang will fix that.
